index.html

```diff
@@ -139,7 +139,7 @@ <h2>Special Thanks</h2>
     <div class="col-md-6" id="books">
        <a href="http://www.packtpub.com/geoserver-share-edit-geospatial-data-beginners-guide/book"><img src="img/books.png"/ ></img></a>
        <p class="text-left">
-         <a href="https://www.packtpub.com/application-development/geoserver-beginner’s-guide">GeoServer Beginner's Guide</a>, <a href="https://www.packtpub.com/application-development/geoserver-cookbook">GeoServer Cookbook</a>, <a href="https://www.packtpub.com/networking-and-servers/mastering-geoserver">Mastering GeoServer</a>, <a href="https://www.packtpub.com/application-development/geoserver-beginner%E2%80%99s-guide-second-edition">GeoServer Beginner's Guide - Second Edition</a> <a href="https://www.packtpub.com/application-development/fundamentals-geoserver-video">Fundamentals of Geoserver [Video]</a> and <a href="https://www.packtpub.com/application-development/expert-geoserver">Expert Geoserver</a> are available from Packt Publishing.
+         <a href="https://www.packtpub.com/application-development/geoserver-beginner’s-guide">GeoServer Beginner's Guide</a>, <a href="https://www.packtpub.com/application-development/geoserver-cookbook">GeoServer Cookbook</a>, <a href="https://www.packtpub.com/networking-and-servers/lting-geoserver">Mastering GeoServer</a>, <a href="https://www.packtpub.com/application-development/geoserver-beginner%E2%80%99s-guide-second-edition">GeoServer Beginner's Guide - Second Edition</a> <a href="https://www.packtpub.com/application-development/fundamentals-geoserver-video">Fundamentals of Geoserver [Video]</a> and <a href="https://www.packtpub.com/application-development/expert-geoserver">Expert Geoserver</a> are available from Packt Publishing.
```

Comment:
I don't think we wanted to substitute for 'master' here.

Reply:
Good catch